zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -369,4 +388,31 @@ export function PrevArrow(props) {
   );
 }
 
+export function EnlargedImgArrow(props) {
+  const { className, style, onClick } = props;
+  return (
+    <div
+      className={className}
+      style={{ ...style, 
```

Comment:
Better to use class for this and FYI the style/any prop spread should be at the end so we can override styles